main/player.py

```diff
@@ -22,20 +23,17 @@ def add_to_backpack(self, item):
     def remove_from_backpack(self, item):
         if item in self.backpack:
             self.backpack.remove(item)
-            print(f"{item} removed from backpack")
-        else:
-            print(f"{item} not found in backpack")
-
-    def expand_backpack(self):
-        self.backpack_size += 1
-        print(f"Backpack size has increased to {self.backpack_size}")
 
-    def gain_experience(self, gained_experience):
-        self.experience += gained_experience
-        if self.experience >= self.experience_needed:
+    def gain_experience(self, sum_of_enemys_stats):
+        self.experience += sum_of_enemys_stats
+        while self.experience >= self.experience_needed:
+            self.experience -= self.experience_needed
             self.level_up()
 
     def level_up(self):
         self.level += 1
         self.experience_needed = int(self.experience_needed * 1.2)
-        print(f"Your actual level: {self.level}")
\ No newline at end of file
+        self.strength += 2
+        self.defense += 2
+        self.health_points_max += 10
```

Comment:
IMO we should add reference for config.ini. In the future it'll be easier to balance game by changing values in just one place.

Reply:
You mean things that are changed while we level up? It is only in one place (player's method)

Reply:
Yes, but I think it will be much easier to save all hardcoded values into config.ini.. Imo 
during rebalancing it will be easier to search rights value in config than.scroling.threw Class character and looking for place to change it. I think that should be one of a topic in next meeting.

Reply:
You were right. This way its easier to balance :) Thanks for tips!